lib/DateTime.pm

```diff
@@ -1531,6 +1536,7 @@ sub subtract_datetime {
     );
 }
 
+## no critic (Subroutines::ProhibitManyArgs)
```

Comment:
No, all these `no critic` things needs to come before the code that critic complains about.